bot/cogs/bot.py

```diff
@@ -103,8 +103,9 @@ def codeblock_stripping(self, msg: str):
                 return content
 
     async def on_message(self, msg: Message):
-        if msg.channel.id in self.previous_format_times:
-            if time.time()-self.previous_format_times[msg.channel.id] > 300 or msg.channel.id == DEVTEST_CHANNEL:
+        if msg.channel.id in self.previous_hint_times:
+            on_cooldown = time.time()-self.previous_hint_times[msg.channel.id] > 300
```

Comment:
that doesn't make sense, if `time.time()-self.previous_hint_times[msg.channel.id] > 300` is True then this is **not** on cooldown. you should flip the logic in this expression so that the result is False if it's not cooling down, then use a `not` in the next line like in my example. Also, `-` should be surrounded by spaces.